### PR TITLE
[TDV-810] Documents manager

### PR DESCRIPTION
**Pull request by ardier16**

### CODE REVIEW

File: src/api2/managers/documents-manager.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  }
+
+  async _postFormData (formData) {
+    const config = {
+      headers: {
+        [HEADER_CONTENT_TYPE]: MIME_TYPE_MULTIPART_FORM_DATA
+      }
+    }
+
+    await this._axios.post(this._storageURL, formData, config)
```
Comment (max-nes) [inline src/api2/managers/documents-manager.js]: add an error parser that throws a new "FileUploadError" (the name is an example)

File: src/api2/managers/documents-manager.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+import { ApiCaller } from '../api-caller'
+
+const HEADER_CONTENT_TYPE = 'Content-Type'
+const MIME_TYPE_MULTIPART_FORM_DATA = 'multipart/form-data'
+
+/**
+ * DocumentsManager uploads a document to the storage server
+ */
+export class DocumentsManager {
```
Comment (max-nes) [inline src/api2/managers/documents-manager.js]: consider adding a method to get a file
or to build the url to get it

File: src/api2/managers/documents-manager.js (hunk 1 above)
Comment (ardier16) [reply src/api2/managers/documents-manager.js]: Added

File: src/api2/managers/documents-manager.js (hunk 2 above)
Comment (ardier16) [reply src/api2/managers/documents-manager.js]: Added

**Pull request merged**